### PR TITLE
#481 Review and adjust access right settings

### PR DESCRIPTION
**Pull request by nakometal**
## Description

* Disable standard `action_done` button.
* Added custom `action_done` button that displays when operated by Manager and Users.  

## Module

**stock_view_adj**

### CODE REVIEW

File: stock_view_adj/views/stock_inventory_views.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <field name="name">stock.inventory.form</field>
+        <field name="model">stock.inventory</field>
+        <field name="inherit_id" ref="stock.view_inventory_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//form/header/button[@name='action_done']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//form/header/button[@name='prepare_inventory']" position="after">
+                <button name="action_done" states="confirm" string="Validate Inventory" type="object" class="oe_highlight"/>
+            </xpath>
```
Comment (yostashiro) [inline stock_view_adj/views/stock_inventory_views.xml:12]: @nakometal Can you try and see if this works, instead?
```
            <xpath expr="//form/header/button[@name='action_done']" position="attributes">
                <!-- remove stock.group_stock_manager assignment-->
                <attribute name="groups_id"></attribute>
            </xpath>
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
@nakometal Let's also adjust the following points in the manufest file:
- bump `version` to `10.0.1.1.0`
- change `website` to `https://www.quartile.co`
- change `name` to just "Stock View Adjust" or something like that.  Would be good to add a couple of lines of explanation to `description` instead.

File: stock_view_adj/views/stock_inventory_views.xml (hunk 1 above)
Comment (nakometal) [reply stock_view_adj/views/stock_inventory_views.xml:12]: @yostashiro It worked If it changes the attribute name `groups_id` to  `groups`.

Comment (nakometal) [reply stock_view_adj/views/stock_inventory_views.xml:12]: <xpath expr="//form/header/button[@name='action_done']" position="attributes">
                
                <attribute name="groups"></attribute>
            </xpath>

**Pull request merged**